### PR TITLE
Feat: remove doposit-event field

### PR DESCRIPTION
**Pull request by mrLSD**
## Description

`Fee` ss unused field in `deposit-event` softly removed.
#31 

Gas cost and tests was not changed.

### CODE REVIEW

File: eth-connector/src/deposit_event.rs [hunk 1]
```diff
@@ -260,6 +259,7 @@ impl DepositedEvent {
                 kind: ParamType::Uint(256),
                 indexed: false,
             },
+            // NOTICE: fee field is currently unused
```
Comment (aleksuss) [inline eth-connector/src/deposit_event.rs:262]: But should we have this event parameter anyway? Is it some kind of standard?

Comment (mrLSD) [reply eth-connector/src/deposit_event.rs]: It was wrong comment. Deleted

File: eth-connector-tests/src/connector.rs [hunk 2]
```diff
@@ -528,7 +527,7 @@ async fn test_deposit_with_0x_prefix() -> anyhow::Result<()> {
         data: ethabi::encode(&[
             ethabi::Token::String(message),
             ethabi::Token::Uint(U256::from(deposit_event.amount)),
-            ethabi::Token::Uint(U256::from(deposit_event.fee.as_u128())),
+            ethabi::Token::Uint(U256::from(fee.as_u128())),
```
[the remark this answers is not in the document]
Comment (aleksuss) [reply eth-connector-tests/src/connector.rs:530]: And also [here](https://github.com/aurora-is-near/aurora-eth-connector/blob/master/eth-connector/src/deposit_event.rs#L297) ?

File: eth-connector/src/deposit_event.rs [hunk 3]
```diff
@@ -237,7 +237,6 @@ pub struct DepositedEvent {
     pub sender: Address,
     pub token_message_data: TokenMessageData,
     pub amount: Balance,
-    pub fee: Fee,
```
Comment (aleksuss) [inline eth-connector/src/deposit_event.rs:240]: But you left 
```rust
EventParam {
    name: "fee".to_string(),
    kind: ParamType::Uint(256),
    indexed: false,
},
```
Is it intentional?

File: eth-connector-tests/src/connector.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (mrLSD) [reply eth-connector-tests/src/connector.rs:530]: It used for `TokenMessageData` and sent to Engine for `ft_on_transfer`

File: eth-connector/src/deposit_event.rs (hunk 3 above)
Comment (mrLSD) [reply eth-connector/src/deposit_event.rs:240]: It's EthEvent field used for `TokenMessageData` for `ft_on_transfer`

**Review by aleksuss — APPROVED** [review]

**Pull request merged**